packages/ag-charts-website/src/content/docs/error-bars-test/_examples/temperatures/main.ts

```diff
@@ -112,34 +117,45 @@ function randomDelta(min: number, max: number) {
 
 function randomiseData() {
     if (options.series !== undefined) {
-        const fn = (value: Datum, index: number): Datum => {
+        const fn = (value: Datum): Datum => {
             const delta = randomDelta(-4, 4);
-            return {
-                month: value.month,
-                temperature: value.temperature + delta,
-                temperatureLower: value.temperatureLower + delta,
-                temperatureUpper: value.temperatureUpper + delta,
-            };
+            const { month, temperature, temperatureLower, temperatureUpper } = value;
+            if (temperatureLower === undefined || temperatureUpper === undefined) {
+                return {
+                    month,
+                    temperature: temperature + delta,
+                };
+            } else {
+                return {
+                    month,
+                    temperature: temperature + delta,
+                    temperatureLower: temperatureLower + delta,
+                    temperatureUpper: temperatureUpper + delta,
+                };
+            }
         };
-        options.series[0].data = getData().map(fn);
-        options.series[1].data = getData2().map(fn);
+        options.series[0].data = options.series[0].data?.map(fn);
```

Comment:
Nit: We need to avoid optional chaining in examples, as it generates a lot of boilerplate (see the Javascript version of examples).

Reply:
Does this guidance apply to test pages too? My understanding is that the `error-bars-test` is only meant for us.

Reply:
Sorry, my mistake 👍